### PR TITLE
Converted recurly_hosted to D8.

### PR DESCRIPTION
**Pull request by markdorison**
@CHROMATIC-LLC/recurly Ready for review before I post to D.O.

### CODE REVIEW

File: modules/recurly_hosted/recurly_hosted.install [hunk 1]
```diff
@@ -9,14 +9,15 @@
...
+  $requirements = [];
   if ($phase === 'install') {
     // Verify that recurlyjs isn't enabled at the same time.
-    if (module_exists('recurlyjs')) {
-      $requirements['recurly_hosted_conflict']['title'] = $t('Recurly.js conflict');
-      $requirements['recurly_hosted_conflict']['description'] = $t('The "Recurly hosted pages" could not be enabled because the Recurly.js module is active. If specifically needing to use Recurly hosted pages, disable Recurly.js first.');
-      $requirements['recurly_hosted_conflict']['severity'] = REQUIREMENT_ERROR;
+    if (\Drupal::moduleHandler()->moduleExists('recurlyjs')) {
+      $requirements['recurly_hosted_conflict'] = [
+        'title' => t('Recurly.js conflict'),
```
Comment (AlannaBurke) [inline modules/recurly_hosted/recurly_hosted.install:17]: Ooh, we don't have to use `get_t` in install files now?

Comment (markdorison) [inline modules/recurly_hosted/recurly_hosted.install:17]: https://www.drupal.org/node/2021435

Comment (AlannaBurke) [inline modules/recurly_hosted/recurly_hosted.install:17]: :tada::tada::tada::tada::tada:

**Comment by AlannaBurke** [comment]
:+1:

**Comment by markdorison** [comment]
Posted to [D.O.](https://www.drupal.org/node/2641558#comment-10705268)

**Pull request closed without merging**